addons/point_of_sale/models/pos_session.py

```diff
@@ -178,6 +178,9 @@ def load_data(self, models_to_load, only_data=False):
                     'error': e.args[0]
                 }
 
+            if 'id' not in response[model]['fields']:
+                response[model]['fields'] += ['id']
+
```

Comment:
This should also be removed, otherwise, most of the models will only have ["id"] as fields-to-load.